shadows/framework/src/main/java/org/robolectric/shadows/ShadowBitmapFactory.java

```diff
@@ -185,18 +187,22 @@ protected static Bitmap decodeByteArray(
     }
 
     Point imageSize = getImageSizeFromStream(new ByteArrayInputStream(data, offset, length));
-    return create(desc, opts, imageSize);
+    return create(desc, null, opts, imageSize);
   }
 
   static Bitmap create(String name) {
     return create(name, null);
   }
 
   public static Bitmap create(String name, BitmapFactory.Options options) {
-    return create(name, options, null);
+    return create(name, null, options, null);
   }
 
-  public static Bitmap create(final String name, final BitmapFactory.Options options, final Point widthAndHeight) {
+  public static Bitmap create(
```

Comment:
If you really want a 4-arg create that is fine, just make it private fo now. I would not prefer not to introduce new public APIs in ShadowBitmapFactory.

Also, can you keep the 3-arg `create(String, final BitmapFactory.Options, Point)`  to avoid breaking exising tests?

Reply:
OK, that's what I'm looking forward to